### ISSUE TITLE
Unexpected compile error: "Cannot reference type: Microsoft.FSharp.Core.Option"

### ISSUE THREAD

**Issue by funlambda** [fsprojects/Fable#285]
I'm having some compilation problems after upgrading to the latest version of Fable (0.4.4). I have been getting this error during compilation in a few places where I'm pattern matching on options and returning float values:

> ERROR: Cannot reference type: Microsoft.FSharp.Core.Option

Here's some F# code that triggers the error:

```fsharp
let test (x: float option) =
   match x with
   | Some y -> y + 3.
   | None -> 0.
```

If I replace `` y + 3 `` with just `` y `` it compiles fine. If I use other float operations (+,/,-) I still get the error.

**Comment by alfonsogarciacaro** [comment]
Thanks for the report, unfortunately this is a problem that surfaced in the latest version because other changes. I've released `fable-compiler@0.4.5` with the fix :+1:

**Comment by funlambda** [comment]
Fantastic -- thanks!
